src/lib/holocene/toaster.svelte

```diff
@@ -13,7 +13,7 @@
   };
 </script>
 
-<div class="fixed bottom-5 right-5 z-50 flex flex-col items-end gap-2">
+<div class="fixed bottom-5 right-5 z-[99999] flex flex-col items-end gap-2">
```

Comment:
Fun fact i just googled, the highest z-index possible is 2147483647. Don't change it to that, cuz that's madness.

Reply:
Ok but why is the minimum z-index −214748364**8** 🙃